smdebug/core/utils.py

```diff
@@ -18,6 +18,15 @@
 from smdebug.exceptions import IndexReaderException
 
 
+def ensure_dir(file_path, is_file=True):
```

Comment:
this seems, buggy. 

Can we remove this "is_file" flag? doesn't seem to be used and makes the code unclear.

Reply:
This change does not seem related to what I want to achieve with my PR.
I can submit a new PR with this change and add you as a reviewer.

Reply:
Double checked, there is no usage of the is_file flag, so removing it, in this PR